klc-cert-manager/config/rendered/release.yaml

```diff
@@ -0,0 +1,357 @@
+apiVersion: v1
```

Comment:
this file should be gitignored, instead the folder should contain an empty `.gitkeep` file

Reply:
git keep is there I will remove the file 👍